1

```diff
@@ -0,0 +1,24 @@
+Reduce calls to CRM_Member_PseudoConstant::membershipType
```

Comment:
@eileenmcnaughton please remove this file

Reply:
dang

Reply:
gone by lunchtime (yours, not mine)